src/domains/pay-wallet-common/00_alerts.tf

```diff
@@ -199,7 +199,7 @@ AzureDiagnostics
 | summarize
     Total=count(),
     Success=countif(responseCode_d < 500 and DurationMs < 350)
-    by Time = bin(TimeGenerated, 15m)
+    by Time = bin(TimeGenerated, 10m)
```

Comment:
Why set 10 minutes for timebin? Didn't we say to trying adjust only traffic threshold for this step?